bigbluebutton-html5/imports/ui/components/UserAvatar /UserAvatar.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
```

Comment:
In this component there is space in folder name `components/UserAvatar /UserAvatar.jsx`, remove the space from folder name

Reply:
fixed